pkg/instancetype/apply/scheduler_test.go

```diff
@@ -0,0 +1,59 @@
+package apply_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
+
+	virtv1 "kubevirt.io/api/core/v1"
+	v1beta1 "kubevirt.io/api/instancetype/v1beta1"
+
+	"kubevirt.io/kubevirt/pkg/instancetype/apply"
+	"kubevirt.io/kubevirt/pkg/libvmi"
+)
+
+var _ = Describe("Apply to vmi - scheduler", func() {
```

Comment:
nit: You could have kept the names of the original Contexts. E.g.

`Apply instancetype.spec.SchedulerName`